platform/ext/target/nordic_nrf/nrf5340/partition/flash_layout.h

```diff
@@ -136,12 +136,12 @@
 
 #define FLASH_PS_AREA_OFFSET            (FLASH_AREA_SCRATCH_OFFSET + \
                                          FLASH_AREA_SCRATCH_SIZE)
-#define FLASH_PS_AREA_SIZE              (0x4000)   /* 16 KB */
+#define FLASH_PS_AREA_SIZE              (0x5000)   /* 20 KB */
```

Comment:
The layout description at the beginning of the file should be updated, too.
And nRF9160 should be updated similarly.

But do we need this change, actually? I saw that not all platforms were using these larger sizes (20 kB and 16 kB) of storage areas. Some, like NXP and STM ones, and Musca-A are still using 16/8 kB.

Reply:
We have space for it, but I can try it out to see if which exact changes are needed to get the tests to run.

Reply:
It wasn't needed so I reverted these changes.